pkgs/development/libraries/glew/default.nix

```diff
@@ -1,6 +1,14 @@
-{ lib, stdenv, fetchurl, fetchpatch, cmake, libGLU, libXmu, libXi, libXext
+{ lib
+, stdenv
+, fetchurl
+, fetchpatch
+, cmake
+, libGLU
+, libXmu
+, libXi
+, libXext
 , OpenGL
-, enableEGL ? false
+, enableEGL ? (!stdenv.isDarwin)
```

Comment:
Note: parens are redundant, and the bulk of Nixpkgs switched to nixmft-rfc-style by now

Reply:
I'll leave that up to the big reformatting :-)

Reply:
Yeah-yeah, let's keep adding noise to the git-blames